scripts/setup_check.sh

```diff
@@ -38,3 +32,13 @@ if ! git rev-parse origin/master >& /dev/null; then
   git remote set-branches --add origin master
   git fetch origin
 fi
+
+# Setup mint which installs a versioned swiftformat.
+set -x
+brew upgrade mint || brew install mint || true
+mint bootstrap --link
+
+brew install clang-format
```

Comment:
AFAIK this command will fail if `clang-format` is already installed. Also I wonder if `clang-format` can be installed with mint as well?

Reply:
I checked. Unfortunately no -but at least clang-format seems more stable ...